kolibri/core/assets/src/vue/session-nav-widget/index.vue

```diff
@@ -28,7 +28,7 @@
               <p id="dropdown-usertype">{{ userkind }}</p>
             </li>
             <li id="logout-tab">
-              <div tabindex="0" @keyup.enter="userLogout" @click="logout" :aria-label="logOutText">
+              <div tabindex="0" @keyup.enter="userLogout" @click="logout" :aria-label="$tr('logOut')">
```

Comment:
@radinamatic is this kosher for aria? It'll just have the same text as the button